protobuf.bzl

```diff
@@ -193,6 +193,30 @@ def cc_proto_library(
       **kargs)
 
 
+def internal_gen_well_known_protos_java(srcs):
+  """Bazel rule to generate the gen_well_known_protos_java genrule
+
+  Args:
+    srcs: the well known protos
+  """
+  root = Label("%s//protobuf_java" % (REPOSITORY_NAME)).workspace_root
```

Comment:
I'm new to bazel so I could be wrong about this but BUILD files aren't python, right, where as bzl files are? I couldn't see how to do either the necessary conditional or the Label call in a BUILD file.

If there is a way to do this? Happy to improve with a bit of direction.